spikeextractors/extraction_tools.py

```diff
@@ -33,6 +33,27 @@ def read_python(path):
     return metadata
 
 
+def write_python(path, dict):
+    '''Saves python dictionary to file
+
+    Parameters
+    ----------
+    path: str or Path
+        Path to save file
+    dict: dict
+        dictionary to save
+    '''
+    with Path(path).open('w') as f:
+        for k, v in dict.items():
+            if isinstance(v ,str) and not v.startswith("'"):
+                if 'path' in k and 'win' in sys.platform:
+                    f.write(str(k) + " = r'" + str(v) + "'\n")
+                else:
+                    f.write(str(k) + " = '" + str(v) + "'\n")
+            else:
+                f.write(str(k) + " = " + str(v) + "\n")
+
+
```

Comment:
Why do we need a home made seriliazer ?

Reply:
this takes care of windows paths. Feel free to change it! but we should test on windows as well

Reply:
this takes care of windows path. Feel free to modify it if you have a better solution